test/compilable/extra-files/json2.out

```diff
@@ -0,0 +1,62 @@
+{
+    "buildInfo": {
+        "config": "VALUE_REMOVED_FOR_TEST",
+        "cwd": "VALUE_REMOVED_FOR_TEST",
+        "importPaths": [
+            "compilable",
+            "..\/..\/druntime\/import",
```

Comment:
how about making sanitize replace `\/` with `/` ?

Reply:
This is the JSON generated by `std.json` (not dmd).  Dmd doesn't generate these weird escaped slashes.  To fix this we would need to post-process the JSON after it is sanitized, basically another JSON parser that sanitizes after `std.json`...lol :)

Reply:
lol indeed... std.json really sux

Reply:
wait, there already is an option for this! https://dlang.org/phobos/std_json.html#.JSONOptions.doNotEscapeSlashes

I'll updated the PR to use it :)